src/__tests__/package_contents.js

```diff
@@ -0,0 +1,133 @@
+import React from "react";
+import PackageContents from "../package_contents";
+import {render, cleanup} from "react-testing-library";
+
+let doc;
+
+const ul_elts = () => doc.baseElement.getElementsByTagName('ul');
+const li_elts = () => ul_elts().item(0).children;
+const li_elt = i => li_elts().item(i);
+const a_elt = i => li_elt(i).getElementsByTagName('a').item(0);
```

Comment:
We'll probably at least want to use one of the built-ins `querySelector`, `querySelectorAll`. Alternatively, we could add like [cheerio](https://cheerio.js.org/) for selection/traversal or [enzyme](https://airbnb.io/enzyme/), which is geared specifically for testing component output.

Reply:
Yeah I'm all for that. I tried to limit myself to the DOM because I didn't want to add "choose the best DOM traversal library" to the scope.

Reply:
Ahh, wise, as usual. We can decide/add something later.